Tests/RxFeedbackTests/FeedbackLoopsTests.swift

```diff
@@ -0,0 +1,212 @@
+//
+//  FeedbackLoopsTests.swift
+//  RxFeedback
+//
+//  Created by Alexander Sokol on 23/09/2017.
+//  Copyright © 2017 Krunoslav Zaher. All rights reserved.
+//
+
+import Foundation
+import XCTest
+import RxFeedback
+import RxSwift
+import RxTest
+
+class FeedbackLoopsTests: XCTestCase {
+    let bag = DisposeBag()
+}
+
+// Tests on the react function with not an equatable or hashable Control.
+extension FeedbackLoopsTests {
+
+    func testIntialNil() {
+        // Prepare
+        let scheduler = TestScheduler(initialClock: 0)
+        let query: (String) -> Void? = { _ in return nil }
```

Comment:
I think we should test non void return value is being passed to the effects lambda.